pkg/manager/manager.go

```diff
@@ -242,7 +262,7 @@ func (m *jobManager) Start() error {
 	return nil
 }
 
-func paramsFromAnnotation(value string) (map[string]string, error) {
+func ParamsFromAnnotation(value string) (map[string]string, error) {
```

Comment:
This method feels like it belongs in a more common/shared place since it's used in multiple packages, but really has nothing to do, specifically, with the `manager`

Reply:
Moved the function to `utils.go`.